muacryptcc/filestore.py

```diff
@@ -62,20 +63,24 @@ def __setitem__(self, key, value):
         # assert key == value.identity()
         self.file_set(key, bdata)
 
+    @property
+    def url(self):
+        return self._dir
```

Comment:
i don't see why you need to add this attribute -- the below "assert store.url" could also just be "store.dir" for now